src/components/LoginDialog.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <q-card style="min-width: 25%">
+    <q-card-section>
+      <div class="text-h6">
+        Log in
+      </div>
+    </q-card-section>
+    <form @submit.prevent="onSubmit">
+      <q-card-section>
+        <q-input
+          v-model="username"
+          dense
+          outlined
+          placeholder="Username"
+          stack-label="Stack Label"
+          :rules="[val => val.length<=15 || 'Usernames should not be over 15 characters']"
```

Comment:
in the database, usernames are limited at 20 characters

Reply:
Ah thank you!! (I wanted to ask but it was so late that I didn't want to send a chat message... and this morning I forgot)